README.md

```diff
@@ -56,10 +56,26 @@ $ stix-shifter translate qradar query {} "[ipv4-addr:value = '127.0.0.1']" {}
 
 **Note:** In order to build `stix-shifter` packages from source follow the below prerequisite steps:
    1. Go to the stix-shifter parent directory
-   2. Generate latest requirements.txt: `python3 generate_requirements.py`
-   3. Install the dependencies in your python 3 environment: `pip install -r requirements.txt` 
-   4. Alternatively you can create a Python 3 virtual environemnt:
-       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && pip install -r requirements-dev.txt`
+   2. Run setup: `python3 setup.py install`
+   3. Alternatively you can create a Python 3 virtual environemnt:
+       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && python3 setup.py install`
```

Comment:
I don't think you need to keep **Note:** section as you are adding a new subsection `### Running From the Source`

Reply:
I will remove Note keyword, but the explanation belongs to this section